docs/users/installation.md

```diff
@@ -382,44 +369,28 @@ session.
 > scalafix RemoveUnused
 ```
 
-The `scalafixEnable` command automatically runs
-`addCompilerPlugin(scalafixSemanticdb)` and `scalacOptions += "-Yrangepos"` for
-all eligible projects in the builds. The change in Scala compiler options means
-the project needs to be re-built on the next `compile`.
+The `scalafixEnable` command automatically enables semanticdb output and adds
+`scalacOptions += "-Yrangepos"` for all eligible projects in the builds. The
+change in Scala compiler options means the project may need to be re-built on
+the next `compile`.
 
 > The `scalafixEnable` command must be re-executed after every `reload` and when
 > sbt shell is exited.
 
-### Optionally enable SemanticDB
```

Comment:
I removed this as it's now fairly standard since https://www.scala-sbt.org/1.x/docs/sbt-1.3-Release-Notes.html#SemanticDB+support